### ISSUE TITLE
Remove "Loading worlds..." message

### ISSUE THREAD

**Issue by viet** [minecraftly/minecraftly#20]
Upon joining, there's a message "Loading worlds..."

I think this message is better off be like "You are now in <username>'s world." Basically showing the world that the player is currently in.

The message will show at join or whenever their world changes

**Comment by SilverCory** [comment]
Yeah, sorry you've asked for this one a few times and it's dropped off my things done. This will be the first thing I change >:D

**Comment by viet** [comment]
Great to know. Thanks!

**Comment by SilverCory** [comment]
On second thoughts we'll have to resolve the name, easily done.. I think.
Delaying this for a day or two.
